### DISCUSSION TITLE
Multiple Rx and Tx

### DISCUSSION THREAD

**Discussion by mmucahitkaya** [svpcom/wfb-ng#222] [Questions]
Hi, can we use Wifibroadcast with multiple RX and TX? And can one raspberry pi be both rx and tx?

**Comment by svpcom** [comment]
Do you mean many drones and one GS case? WFB doesn't support this setup out of box, but can be modified to do that.
RPI can be TX and RX (but in half-duplex mode due to wifi hw limits).

**Reply by svpcom** [reply]
You can ask general questions in https://t.me/EZWBC

**Reply by InanixFR** [reply]
Hi, 

I'm using WFB with one GS (Ubuntu server) with several drones (Raspeberry Pi + RTL8812AU) with some configurations about ports for each

**Reply by mmucahitkaya** [reply]
@Dronotique how can you do that? Can you explain, please?

**Reply by InanixFR** [reply]
Hi,

You can start several instance of WFB by using the environment variable WIFIBROADCAST_CFG that allow you to specify the wfb configuration file to use for each instance (see https://github.com/svpcom/wifibroadcast/discussions/204).
In this configurations files you have to modify each port to be unique on your system. And that is all.
